proactive-jupyter-kernel/pragma.py

```diff
@@ -640,7 +640,15 @@ def is_valid_import_dot(data):
 
 
 def is_valid_submit_job(data):
-    return is_valid_write_dot(data)
+    pattern_name = r"^[a-zA-Z_]\w*$"
```

Comment:
could you please example of good value by this pattern in the comment? thanks

Reply:
Done. Thanks.